### PR TITLE
Fix linting kubectl apply

### PR DESCRIPTION
**Pull request by obitech**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Fixing golint errors for `pkg/kubectl/cmd/apply` 

**Which issue(s) this PR fixes**:
Ref #68026 

**Special notes for your reviewer**:
This is my first PR to the Kubernetes project ever, so please let me know if there is something I didn't do correctly. I'm happy to go back and fix it :)

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by obitech** [comment]
I've signed the CLA

**Comment by obitech** [comment]
I assume the failed test is flaky, will rerun. I'm not that familiar with the whole test suite yet, so please let me know if I missed anything.

/test pull-kubernetes-e2e-gce

**Comment by obitech** [comment]
Not quite sure why the tests keep failing...

```
[k8s.io] [sig-node] Security Context should support container.SecurityContext.RunAsUser And container.SecurityContext.RunAsGroup [LinuxOnly]
```

I haven't touched this, rerunning...

/test pull-kubernetes-e2e-gce

**Comment by obitech** [comment]
All tests are passing, assigning additional approver for `hack/`:

/assign @soltysh

**Comment by obitech** [comment]
This PR has been open for almost two months now, all the tests are passing. Is there anything I can do to speed up the process?

**Comment by dims** [comment]
/milestone v1.15

@obitech i added the milestone, so hopefully reviewers will make a call during this release cycle

**Comment by obitech** [comment]
@dims, awesome thanks you! :)

File: pkg/kubectl/cmd/apply/apply_edit_last_applied.go [hunk 1]
```diff
@@ -55,6 +55,7 @@ var (
 		kubectl apply edit-last-applied -f deploy.yaml -o json`)
 )
 
+// NewCmdApplyEditLastApplied created the cobra CLI command for the `apply edit-last-applied` command.
```
Comment (SataQiu) [inline pkg/kubectl/cmd/apply/apply_edit_last_applied.go]:
```suggestion
// NewCmdApplyEditLastApplied creates the cobra CLI command for the `apply edit-last-applied` command.
```

File: pkg/kubectl/cmd/apply/apply_set_last_applied.go [hunk 2]
```diff
@@ -37,6 +37,7 @@ import (
 	"k8s.io/kubernetes/pkg/kubectl/util/templates"
 )
 
+// SetLastAppliedOptions defines options for the `apply set-last-applied` command.`
```
Comment (SataQiu) [inline pkg/kubectl/cmd/apply/apply_set_last_applied.go]:
```suggestion
// SetLastAppliedOptions defines options for the `apply set-last-applied` command.
```

File: pkg/kubectl/cmd/apply/apply_set_last_applied.go [hunk 3]
```diff
@@ -81,13 +83,15 @@ var (
...
 
+// NewSetLastAppliedOptions takes option arguments from a CLI stream and returns it at SetLastAppliedOptions type.
 func NewSetLastAppliedOptions(ioStreams genericclioptions.IOStreams) *SetLastAppliedOptions {
 	return &SetLastAppliedOptions{
 		PrintFlags: genericclioptions.NewPrintFlags("configured").WithTypeSetter(scheme.Scheme),
 		IOStreams:  ioStreams,
 	}
 }
 
+// NewCmdApplySetLastApplied creates the cobra CLI `apply` subcommand `set-last-applied`.`
```
Comment (SataQiu) [inline pkg/kubectl/cmd/apply/apply_set_last_applied.go]:
```suggestion
// NewCmdApplySetLastApplied creates the cobra CLI `apply` subcommand `set-last-applied`.
```

File: pkg/kubectl/cmd/apply/apply_view_last_applied.go [hunk 4]
```diff
@@ -31,6 +31,7 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
+// ViewLastAppliedOptions defines options for the `apply view-last-applied` command.`
```
Comment (SataQiu) [inline pkg/kubectl/cmd/apply/apply_view_last_applied.go]:
```suggestion
// ViewLastAppliedOptions defines options for the `apply view-last-applied` command.
```

File: pkg/kubectl/cmd/apply/apply_view_last_applied.go [hunk 5]
```diff
@@ -65,6 +67,7 @@ func NewViewLastAppliedOptions(ioStreams genericclioptions.IOStreams) *ViewLastA
 	}
 }
 
+// NewCmdApplyViewLastApplied creates the cobra CLI `apply` subcommand `view-last-applied`.`
```
Comment (SataQiu) [inline pkg/kubectl/cmd/apply/apply_view_last_applied.go]:
```suggestion
// NewCmdApplyViewLastApplied creates the cobra CLI `apply` subcommand `view-last-applied`.
```

**Comment by obitech** [comment]
@SataQiu thank you for the suggestions!

File: pkg/kubectl/cmd/apply/apply.go [hunk 6]
```diff
@@ -56,7 +56,8 @@ import (
 	"k8s.io/kubernetes/pkg/kubectl/validation"
 )
 
-type ApplyOptions struct {
```
Comment (prksu) [inline pkg/kubectl/cmd/apply/apply.go]: don't change these.

**Review by ahmadnurus — COMMENTED** [review]
renaming `*Options` to `Options` has been rejected in the past.

File: pkg/kubectl/cmd/apply/apply.go (hunk 6 above)
Comment (SataQiu) [reply pkg/kubectl/cmd/apply/apply.go]: Yes, these changes are not recommended, just leave them as they are.
Others LGTM.

**Review by obitech — COMMENTED** [review]
I have changed it back with the recent commit

**Comment by obitech** [comment]
@SataQiu @prksu would it work like this ?

**Comment by timmycarr** [comment]
Hi! Friendly reminder from your release team: we are starting the code freeze for 1.15 tomorrow EOD. Just checking in to see if this is still planned for the 1.15 cycle?

**Comment by liggitt** [comment]
if it makes the milestone, that's fine, but it is not required

**Pull request merged**